contents/come-funziona/come-funziona.yml

```diff
@@ -96,7 +94,7 @@ timeline:
       text: L’Unione Europea approva un piano da €750 miliardi che ha l'obiettivo di rilanciare l'economia europea dopo la pandemia di COVID-19 e di renderla più verde e digitale.
     - id: 2
       active: false
-      time: MAGGIO 2021
+      time: Aprile 2021
```

Comment:
2021?

Reply:
![image](https://user-images.githubusercontent.com/52462767/159932394-d167bcc0-b75a-4cd8-82ee-e7522ff033a4.png)

in figma we have "Aprile 2021" in the card "Italia Domani"